docs/DeveloperGuide.adoc

```diff
@@ -800,11 +810,35 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |new user |register and account with NUS net ID |join the module community
+
+|`* * *` |new user |fill in my full name and phone number (strictly 8 numbers) |provide necessary information for the platform maintenance
+
+|`* * *` |new user |fill in optional fiels such as faculty, year of study etc |help my classmates know be better
+
+|`* * *` |user |add contacts |have more classmates to communicate with
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |find a person by NUS NET ID |locate details of persons without having to go through the entire list
```

Comment:
Do you remember your classmate NUSNET ID? Also, instead of using "person" may be "classmate" is better.